xlators/features/shard/src/shard.c

```diff
@@ -1013,7 +1013,7 @@ shard_common_resolve_shards(call_frame_t *frame, xlator_t *this,
 {
     int i = -1;
     uint32_t shard_idx_iter = 0;
-    char path[PATH_MAX] = {
+    char path[sizeof(GF_SHARD_DIR) + GF_UUID_BUF_SIZE + 16] = {
```

Comment:
Defining this length as a constant would be better.

Reply:
Thought this will bit informative. Yes. Making it a macro